src/main/java/com/commercetools/project/sync/CliRunner.java

```diff
@@ -176,10 +197,13 @@ private static Options buildCliOptions() {
     final String syncOptionValue = commandLine.getOptionValue(SYNC_MODULE_OPTION_SHORT);
     final String runnerNameValue = commandLine.getOptionValue(RUNNER_NAME_OPTION_SHORT);
     final boolean isFullSync = commandLine.hasOption(FULL_SYNC_OPTION_SHORT);
+    final boolean syncProjectSyncCustomObjects =
```

Comment:
Use 'is' prefix as variable name so that developer can easier to identify it is a boolean.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/180/commits/d9fa4a9f6aa4869cbff74e6c4c12ac1b1cb5a4c3